dango/testing/tests/proposal_pyth.rs

```diff
@@ -0,0 +1,84 @@
+use {
+    core::time,
+    dango_app::PythProposalPreparer,
+    dango_testing::{setup_test, TestAccount},
+    dango_types::{
+        account_factory::Username,
+        oracle::{ExecuteMsg, PriceSource, PythId, QueryPriceSourcesRequest, QueryPricesRequest},
+    },
+    grug::{btree_map, Addr, Addressable, Coins, Denom, ResultExt},
+    std::{str::FromStr, thread::sleep},
+};
+
+#[test]
+fn proposal_pyth() {
+    let (mut suite, mut account, _, contracts) = setup_test();
+
+    let price_ids = btree_map!(
+        Denom::from_str("usdc").unwrap() => PriceSource::Pyth { id:  PythId::from_str("0xeaa020c61cc479712813461ce153894a96a6c00b21ed0cfc2798d1f9a9e9c94a").unwrap(), precision: 6 },
+        Denom::from_str("btc").unwrap() => PriceSource::Pyth { id: PythId::from_str("0xc9d8b075a5c69303365ae23633d4e085199bf5c520a3b90fed1322a0342ffc33").unwrap(), precision: 8 },
+        Denom::from_str("eth").unwrap() => PriceSource::Pyth { id: PythId::from_str("0xff61491a931112ddf1bd8147cd1b641375f79f5825126d665480874634fd0ace").unwrap(), precision: 18 },
+    );
```

Comment:
This part of the code is so ugly. rustfmt doesn't format macro invocations, so we need to format them manually.

For consistency, with the `btree_map!` and `btree_set!` macros, let's always use the curly braces `btree_map! { /* ... */}` instead of round braces `btree_map!(/* ... */)`.